src/components/staking/action/Deposit.tsx

```diff
@@ -70,7 +70,7 @@ export function Deposit() {
           updateValidators();
           updateStakingDerive();
         }}
-        initialValues={{ stash: stashAccount, promiseMonth: duration, accept: false }}
+        initialValues={{ stash: stashAccount || undefined, promiseMonth: duration, accept: false }}
```

Comment:
Validate in FormModal component, set to undefined if needed

Reply:
resolved